lnwallet/channel.go

```diff
@@ -8679,7 +8680,9 @@ func (lc *LightningChannel) CalcFee(feeRate chainfee.SatPerKWeight) btcutil.Amou
 //
 // NOTE: This should only be used for channels in which the local commitment is
 // the initiator.
-func (lc *LightningChannel) MaxFeeRate(maxAllocation float64) chainfee.SatPerKWeight {
+func (lc *LightningChannel) MaxFeeRate(
+	maxAllocation float64) (chainfee.SatPerKWeight, float64) {
```

Comment:
Please add to the godoc of this stating that the second return value is a fraction value [0,1] representing the share of the total balance that is allocated to fees. I would not ordinarily be able to ascertain this without inspecting the definition.